spk/pyload/src/dsm-control.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Package
+PACKAGE="pyload"
+DNAME="pyLoad"
+
+PIDFILE="/var/run/pyload.pid"
```

Comment:
Move the pidfile to `${INSTALL_DIR}/var/pyload.pid`, we want packages to be self-contained as much as possible.